### PR TITLE
lab 3 notes

### PR DESCRIPTION
**Pull request by rmartin-sc**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,10 +115,9 @@ <h3>My Favourite Youtube Video</h3>
             <!--Found me!-->
 
             <picture>
-                <source type="image/svg" srcset="images/sc-logo.svg">
+                <source type="image/svg+xml" srcset="images/sc-logo.svg">
```
Comment (rmartin-sc) [inline index.html:118]: SVG mime type is "image/svg+xml"

File: index.html [hunk 2]
```diff
@@ -115,10 +115,9 @@ <h3>My Favourite Youtube Video</h3>
...
                 <source type="image/png" srcset="images/sc-logo.png">
-                <img src="images/sc-logo.png" srcset="images/sc-logo.svg 70w" alt="Sault College Logo"
-                    sizes="(max-width:105px) 105px, (max-width:140px) 140px">
+                <img src="images/sc-logo.png" srcset="images/sc-logo.png, images/sc-logo@1.5x.png 1.5x, images/sc-logo@2x.png 2x" alt="Sault College Logo" width="70">
             </picture>
```
Comment (rmartin-sc) [inline index.html:121]: For resolution switching, no sizes attribute is required.  You only need a srcset, specifying screen densities for each source

File: links.html [hunk 3]
```diff
@@ -54,5 +54,6 @@ <h1>Useful Links</h1>
     </footer>
 
         <a href="#top" title="Back to top">◤</a>
+    </div>
```
Comment (rmartin-sc) [inline links.html:57]: Cat ran across your keyboard at an inopportune time?

Comment (JayCWL) [reply links.html:57]: I actually have no idea why that was there, I'm so sorry I missed it.

**Comment by rmartin-sc** [comment]
To be clear, you want​ that there.  It's the closing tag for the width-wrap div.
________________________________

**Pull request merged**